doc/web-doc/.gitignore

```diff
@@ -0,0 +1,12 @@
+# LC_COLLATE=C sort .gitignore
+!.env.example
```

Comment:
Using `LC_COLLATE=C sort .gitignore` as above moves this line around.

Reply:
Luckily we can remove this exception for this project. I'm not sure how we would fix it otherwise.